src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.BetDaoImpl;
+import core.basesyntax.model.Bet;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    BetDao betDao = new BetDaoImpl();
+
```

Comment:
how do you save users into db?

Reply:
in my BetDaoImpl I have method add, which add the bet to the storage

Reply:
yes, BUT: you also have User.class. where is it used?

Reply:
There no requirements in task to do this, but ok, I will add it)